sections/image-with-text.liquid

```diff
@@ -14,7 +14,11 @@
   }
 {%- endstyle -%}
 
-<div class="image-with-text image-with-text--{{ section.settings.content_layout }} page-width isolate{% if settings.text_boxes_border_thickness > 0 and settings.text_boxes_border_opacity > 0 and settings.media_border_thickness > 0 and settings.media_border_opacity > 0 %} collapse-borders{% endif %}{% unless section.settings.color_scheme == 'background-1' and settings.media_border_thickness > 0 and settings.text_boxes_shadow_opacity == 0 and settings.text_boxes_border_thickness == 0 or settings.text_boxes_border_opacity == 0 %} collapse-corners{% endunless %} section-{{ section.id }}-padding">
+{%- if settings.scroll_toggle -%}
+  <script src="{{ 'animations.js' | asset_url }}" id="AnimationsScript-{{ section.id }}"  defer="defer"></script>
```

Comment:
Question: I'm actually not sure this needs to be its own file. What do you think about adding it to `global.js`? We don't need to call it unless the setting is enabled and the file size is small.

Reply:
I didn't realise we could get access to the setting in our scripts. How do I do that?